qcelemental/models/v2/procedures.py

```diff
@@ -348,11 +354,17 @@ def convert_v(
         if check_convertible_version(version, error="TorsionDriveResult") == "self":
             return self
 
+        opthist_class = next(iter(self.optimization_history.values()))[0].__class__
```

Comment:
Nit: Can the `opthist_class` variable be moved into the `if`-clause given it seems to only be used there?